src/components/Nav.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+export default class Nav extends Component {
+  render() {
+    return (
+      <div>
+        <nav>
+          <Link to="/">Home</Link>
+          <Link to="/catagories">Calculator</Link>
```

Comment:
- [ ] I believe the title for this link at line 10 should be Categories and not Calculator. Please fix that 👍